dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/java_gateway.py

```diff
@@ -75,6 +78,21 @@ def __init__(
         auto_convert: Optional[bool] = True,
     ):
         self.java_gateway = launch_gateway(address, port, auto_convert)
+        gateway_version = self.get_gateway_version()
+        if gateway_version is not None and gateway_version != __version__:
+            logger.critical(
+                f"Using unmatched version of pydolphinscheduler (version {__version__}) "
+                f"and Java gateway (version {gateway_version}) may cause errors. "
+                "We strongly recommend you to find the matched version "
+                "(check: https://pypi.org/project/apache-dolphinscheduler)"
+            )
```

Comment:
BTW, can you add this check to our integration test? https://github.com/apache/dolphinscheduler/blob/5289b09817396046489b28857ad54a85c5ffddb2/dolphinscheduler-python/pydolphinscheduler/tests/integration/test_java_gateway.py#L53

Reply:
Sorry I didn't get the point. Assert the `getGatewayVersion` method exists? Or assert the java gateway version `= what`? Please further point it out. thanks

Reply:
@HeZean Sorry for reply later. we should check whether the version is equal or not in our CI. and also test print critical log in our test too.